Anchor.toml

```diff
@@ -20,4 +20,4 @@ program = "target/deploy/mpl_auction_house.so"
 members = [
   "token-metadata/program",
   "auction-house/program"
-] 
+] 
```

Comment:
Maybe not necessary, but I'd have expected that you'd add nft-packs here?
Does `anchor build` work and create the `target/idl/nft_packs.json`?

Reply:
NFT-packs isn't an anchor program so it will not generate idl file